circle.yml

```diff
@@ -11,6 +11,8 @@ dependencies:
     - atom -v
     - apm -v
     - apm install
-# test:
+test:
+  pre:
```

Comment:
We've discussed a similar issue before.  IMO it doesn't make sense to override with the default, it's just redundant.  Also, I explicitly do want to lint before testing.  Until there are specs and we truly do want to override default behavior with the atom plugin testing script, I feel like this is a sensible config...